src/style.css

```diff
@@ -0,0 +1,105 @@
+@import url(..\src\reset.css);
+@import url(..\src\normalize.css);
+@import url(..\src\variables.css);
+@import url(..\src\stars.css);
+
+.card {
+  display: flex;
```

Comment:
`flex` is redundant for the card, block element will be displayed on the new line by default